arcsi/api/__init__.py

```diff
@@ -9,3 +9,25 @@
 
 # TODO add routing here eg
 # arcsi.route("/archive", methods=["GET"])(<method_name>)
+
+
+@arcsi.route("show/<string:show_slug>/archive", methods=["GET"])
+def view_show_archive(show_slug):
+    show_query = Show.query.filter_by(archive_lahmastore_base_url=show_slug)
+    show = show_query.first_or_404()
+    if show:
+        show_items = show.items.filter(Item.play_date < datetime.today() - timedelta(days=1)).all()
+        return jsonify(many_item_details_schema.dumps(show_items))
+    else:
+        return make_response("Show not found", 404, headers)
+
+
+@arcsi.route("show/<string:show_slug>/episode/<string:episode_slug>", methods=["GET"])
+def view_episode_archive(show_slug, episode_slug):
+    episode_slug += ".mp3"
+    item_query = Show.query.filter_by(archive_lahmastore_base_url=show_slug).join(Item, Show.items).first().items.filter_by(play_file_name=episode_slug)
```

Comment:
I think we could simplify this by using the associative table for getting the `item_query` we want to execute. 

Something like `Item.query.join(items_shows).join(Show)`

And so this could be all joined together first and do the filtering on it after. Right now it is filter-join-filter.

But this might not work since we currently use `dynamic` setting for `lazy` relationship type instead of `joined` I'm sure though that we could get back an `item_query` without having to join `Item` to `Show.items` directly